lib/openstudio-standards/prototypes/common/objects/Prototype.refrigeration.rb

```diff
@@ -612,11 +612,11 @@ def model_add_typical_refrigeration(model, building_type)
 
         # Add defrost and dripdown schedules
         defrost_sch = OpenStudio::Model::ScheduleRuleset.new(model)
-        defrost_sch.setName('Refrigeration Defrost Schedule')
-        defrost_sch.defaultDaySchedule.setName("Refrigeration Defrost Schedule Default - #{case_type}")
+        defrost_sch.setName("#{ref_case.name} Refrigeration Defrost Schedule")
```

Comment:
IMO 'Refrigeration' (and perhaps even 'Schedule') don't add anything to these schedule names and can make them hard to use in the app. But not a huge deal.

Reply:
shortened the names